utils/build.ps1

```diff
@@ -3154,7 +3154,7 @@ function Build-Installer([Hashtable] $Platform) {
     BundleFlavor = "offline";
     ImageRoot = "$(Get-InstallDir $Platform)\";
     INCLUDE_SWIFT_DOCC = $INCLUDE_SWIFT_DOCC;
-    INCLUDE_NOASSERTS = $IncludeNoAsserts
+    INCLUDE_NOASSERTS = $IncludeNoAsserts;
```

Comment:
Can we change this to `Variants=...` similar to `Platforms`?

Reply:
sure.